### PR TITLE
Refactor service definitions replacing removed classes

### PR DESCRIPTION
**Pull request by delboy1978uk**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
Fixed tickets | #3343

### CODE REVIEW

File: src/Kunstmaan/AdminBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -174,7 +174,7 @@ services:
             - { name: cache.pool }
 
     kunstmaan_admin.form.type.color:
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Resources/config/services.yml]: Both the services can be deleted as the symfony form types are registered by symfony themself. We don't need the extra services anymore.

**Review by acrobat — CHANGES_REQUESTED** [review]
That was indeed an incorrect leftover of v6. Can you target the PR against the 7.0 branch? The cleanup was done on that branch.

And see my comment inline

File: src/Kunstmaan/AdminBundle/Resources/config/services.yml (hunk 1 above)
Comment (delboy1978uk) [reply src/Kunstmaan/AdminBundle/Resources/config/services.yml]: I'll strip them completely out then! Thanks  again!

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @delboy1978uk

**Pull request merged**